diff --git a/dist/vue-i18n-extract.umd.js b/dist/vue-i18n-extract.umd.js
index ca19c7a2c65d835c84eecb1816ae78c4dffe2c40..6157cbfb64cca65ca3b43291e41e49a50409794e 100644
--- a/dist/vue-i18n-extract.umd.js
+++ b/dist/vue-i18n-extract.umd.js
@@ -54,7 +54,7 @@
     let options;
 
     try {
-      const pathToConfigFile = path__default["default"].resolve(process.cwd(), './vue-i18n-extract.config.js'); // eslint-disable-next-line @typescript-eslint/no-var-requires
+      const pathToConfigFile = path__default["default"].resolve(process.cwd(), './vue-i18n-extract.config.cjs'); // eslint-disable-next-line @typescript-eslint/no-var-requires
 
       const configOptions = require(pathToConfigFile);
 
